front/lib/workspace_usage.ts

```diff
@@ -407,7 +407,7 @@ export async function getAssistantsUsageData(
       COUNT(DISTINCT u."id") AS "distinctUsersReached",
       MAX(CAST(ac."createdAt" AS DATE)) AS "lastEdit"
     FROM
-      "mentions" a
+      "agent_messages" a
```

Comment:
not sure I understand this ? What do we get from fetching agent messages instead of mentions ?

Reply:
nvm me, I understand the issue now 👍